recipes/sparrow/all/conanfile.py

```diff
@@ -0,0 +1,112 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build.cppstd import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=2.0"
+
+
+class SparrowRecipe(ConanFile):
+    name = "sparrow"
+    description = "C++20 idiomatic APIs for the Apache Arrow Columnar Format"
+    license = "Apache-2.0"
+    author = "Man Group"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/man-group/sparrow"
+    topics = ("arrow", "apache arrow", "columnar format", "dataframe")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps"
+    exports_sources = (
+        "include/*",
+        "LICENSE",
+        "src/*",
+        "cmake/*",
+        "docs/*",
+        "CMakeLists.txt",
+        "sparrowConfig.cmake.in",
+    )
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "use_date_polyfill": [True, False],
+    }
+    default_options = {"shared": False, "fPIC": True, "use_date_polyfill": False}
+
+    def requirements(self):
+        if self.options.get_safe("use_date_polyfill"):
```

Comment:
```suggestion
        if self.options.use_date_polyfill:
```
`use_date_polyfill` is not removed by `config_options()` or `configure()`. It's better to avoid `get_safe()` unless necessary, since it can hide recipe bugs.

Reply:
Ok, addressed